content/Cloud/key-value-memory.mdx

```diff
@@ -32,27 +32,4 @@ There are two ways to create a new key-value instance:
 
 ## Using Key-Value Memory in Applications
 
-Key-value memory storage can be integrated with your applications using the Agentuity SDK or 
-directly within an agent context. Note that inside agent projects, you will get access to 
-this storage with zero setup.  Just use it and go.
-
-### Using Key-Value Memory in Agents
-
-When working within an agent context, you can access key-value storage directly through the context object:
-
-```javascript
-export default async function Agent(
-  req: AgentRequest,
-  resp: AgentResponse,
-  ctx: AgentContext,
-) {
-  // Store data in key-value memory
-  await ctx.kv.set('my-key', 'my-value');
-  
-  // Retrieve data from key-value memory
-  const value = await ctx.kv.get('my-key');
-  
-  // Delete data from key-value memory
-  await ctx.kv.delete('my-key');
-}
-```
+Key-value memory storage can be integrated with your applications using the Agentuity SDK.
```

Comment:
just remove this line and remove the entire section "Using Key-Value Memory in Applications"